src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -26,7 +29,13 @@ public boolean analyze() {
     boolean success = checkDependencyManagement();
     // don't combine with previous line, we don't want short-circuit evaluation
     success &= checkPluginManagement();
-
+    log.info("\n\n---- Issues found ----\n");
+    if (unManagedDependencyError) { log.warn(requireManagement.unManagedDependencyMessage()); }
```

Comment:
In our style, we don't put if statements on one line like this. So these should be:
```java
if (condition) {
  log.warn(something);
}
```